.github/workflows/py-ingestion-publish.yml

```diff
@@ -41,10 +41,13 @@ jobs:
             twine check dist/*; \
             twine upload -r testpypi dist/*
   publish:
+    needs: [publish-test]
     runs-on: ubuntu-latest
     environment: release
     steps:
       - uses: actions/checkout@v2
+        with:
+          approvers: akash-jain-10, pmbrull, harshach
```

Comment:
Hi @habibrhmn , is this needed? When I configured the environments I already set protection rules with some reviewers in there

Reply:
Can i get the variables. I will add them in the file. @pmbrull

Reply:
But actually, the question is why do we need to add this in this workflow? It feels easier to just configure the environment directly and have the protection rules automatically running for any workflow that aims at `release`

Reply:
hi @habibrhmn , oki np. Then we could just remove those. This is the list we have configured at the environment 
![image](https://user-images.githubusercontent.com/35870520/167419491-46a5786e-3d3d-4d34-8e7c-c3d64b9d922c.png)